shared/wallets/wallet/settings/container.js

```diff
@@ -22,13 +22,15 @@ const mapStateToProps = (state, {routeProps}) => {
     Constants.getDisplayCurrencyWaitingKey(accountID)
   )
   const saveCurrencyWaiting = anyWaiting(state, Constants.changeDisplayCurrencyWaitingKey)
+  const mobileOnlyMode = Constants.getMobileOnly(state, accountID)
```

Comment:
you can just reach into state here and not use some helper that is only used here